.github/workflows/windows_angle_wheels.yml

```diff
@@ -20,7 +20,7 @@ env:
 
 jobs:
   prepare_angle:
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
I think there's a way to select the specific version of Visual studio to use, without having to force the OS version.

Reply:
Unfortunately looks like this is not feasible on `windows-*` runners 😔 (unlike the macOS ones, which have both VS 2019 and VS 2022 installed)

From `https://github.com/actions/runner-images/issues/3949`:

> Our policy is installing one VS version per platform. On Windows Server 2022, we install only VS 2022 (it is in Preview for now but will be released in future).If your use-case requires using VS 2019, continue using Windows Server 2019 image. We don't have plans to deprecate it in near future.